components/builder-api/src/server/error.rs

```diff
@@ -235,5 +235,5 @@ impl From<string::FromUtf8Error> for Error {
 }
 
 impl From<actix_web::error::BlockingError> for Error {
-    fn from(err: actix_web::error::BlockingError) -> Error { err.into() }
+    fn from(_: actix_web::error::BlockingError) -> Error { Error::System }
```

Comment:
I'm not sure why this `from` was implemented differently before, but can we follow the pattern we use for the other error types and create an `Error:BlockingError(actix_web::error::BlockingError)`? This way we don't lose the original error.